core/jazz_ui/src/app/secondary-components/advanced-filters/OSS/advanced-filters.component.ts

```diff
@@ -84,6 +84,18 @@ export class AdvancedFiltersComponentOSS implements OnInit {
         this.periodSelected=this.periodList[0];
     }
 
+    setSlider(event){
```

Comment:
I think setSlider() is not being used as slider component is removed. Can you remove this method?

Reply:
i removed it